db/controller.js

```diff
@@ -18,7 +18,8 @@ exports.getTopics = (req, res, next) => {
 };
 
 exports.getArticles = (req, res, next) => {
-  fetchAllArticles()
+  const query = req.query;
```

Comment:
ideally, your controller will parse the query to retrieve values before it gets to the model; we're trying to shield our model -- the point of contact with our database -- as much as possible